tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -156,15 +156,15 @@ exports[`test hug.js 1`] = `
   {__DEV__
     ? this.renderDevApp()
     : <div>
-      {routes.map(route => (
-        <MatchAsync
-          key={\`\${route.to}-async\`}
-          pattern={route.to}
-          exactly={route.to === \"/\"}
-          getComponent={routeES6Modules[route.value]}
-        />
-      ))}
-    </div>}
+        {routes.map(route => (
+          <MatchAsync
+            key={\`\${route.to}-async\`}
+            pattern={route.to}
+            exactly={route.to === \"/\"}
+            getComponent={routeES6Modules[route.value]}
+          />
+        ))}
+      </div>}
```

Comment:
Hah, hey @lydell ! Good to see you here from coffeescript6 discussions. 

Would love your thoughts on an alt-js lang I'm working on if you have some time to chat? rattray.alex@gmail.com 

(apologies all for the sidechat)